pkg/controller/controlplane/valuesprovider.go

```diff
@@ -662,12 +662,15 @@ func getIPAMChartValues(
 ) (map[string]interface{}, error) {
 	mode := "ipv4"
 	primaryIPFamily := "ipv4"
-	if networkingConfig := cluster.Shoot.Spec.Networking; networkingConfig != nil {
-		if len(networkingConfig.IPFamilies) == 2 {
-			mode = "dual-stack"
-			primaryIPFamily = strings.ToLower(string(cluster.Shoot.Spec.Networking.IPFamilies[0]))
-		} else if slices.Contains(networkingConfig.IPFamilies, v1beta1.IPFamilyIPv6) {
-			mode = "ipv6"
+	condition := gardencorev1beta1helper.GetCondition(cluster.Shoot.Status.Constraints, "ToDualStackMigration")
```

Comment:
Could you please add a comment to use the constant from gardener/gardener once it is available?